themes/default-bootstrap/modules/blocknewsletter/blocknewsletter.tpl

```diff
@@ -24,18 +24,19 @@
 *}
 <!-- Block Newsletter module-->
 <div id="newsletter_block_left" class="block">
-	<h4>{l s='Newsletter' mod='blocknewsletter'}</h4>
-	<div class="block_content">
-		<form action="{$link->getPageLink('index', null, null, null, false, null, true)|escape:'html':'UTF-8'}" method="post">
-			<div class="form-group{if isset($msg) && $msg } {if $nw_error}form-error{else}form-ok{/if}{/if}" >
-				<input class="inputNew form-control grey newsletter-input" id="newsletter-input" type="text" name="email" size="18" value="{if isset($msg) && $msg}{$msg}{elseif isset($value) && $value}{$value}{else}{l s='Enter your e-mail' mod='blocknewsletter'}{/if}" />
+    <h4>{l s='Newsletter' mod='blocknewsletter'}</h4>
+    <div class="block_content">
+        <form action="{$link->getPageLink('index', null, null, null, false, null, true)|escape:'html':'UTF-8'}" method="post">
+            <div class="form-group{if isset($msg) && $msg } {if $nw_error}form-error{else}form-ok{/if}{/if}" >
+                <input class="inputNew form-control grey newsletter-input" id="newsletter-input" type="text" name="email" size="18" value="{if isset($msg) && $msg}{$msg}{elseif isset($value) && $value}{$value}{else}{l s='Enter your e-mail' mod='blocknewsletter'}{/if}" />
                 <button type="submit" name="submitNewsletter" class="btn btn-default button button-small">
                     <span>{l s='Ok' mod='blocknewsletter'}</span>
                 </button>
-				<input type="hidden" name="action" value="0" />
-			</div>
-		</form>
-	</div>
+                {hook h='displayGDPRConsent' mod='psgdpr' id_module=$id_module}
```

Comment:
As said in https://github.com/PrestaShop/blocknewsletter/pull/29/files#r189231971:

> What if another (native or community) module wants to use this hook? Do you really think that's a good idea to limit that call to a specific module?